src/Network/CurlResult.php

```diff
@@ -229,57 +229,43 @@ private function checkInfo()
 		}
 	}
 
-	/**
-	 * Gets the Curl Code
-	 *
-	 * @return string The Curl Code
-	 */
+	/** {@inheritDoc} */
 	public function getReturnCode()
 	{
 		return $this->returnCode;
 	}
 
-	/**
-	 * Returns the Curl Content Type
-	 *
-	 * @return string the Curl Content Type
-	 */
+	/** {@inheritDoc} */
 	public function getContentType()
 	{
 		return $this->contentType;
 	}
 
-	/**
-	 * Returns the Curl headers
-	 *
-	 * @param string $field optional header field. Return all fields if empty
-	 *
-	 * @return string the Curl headers or the specified content of the header variable
-	 */
-	public function getHeader(string $field = '')
+	/** {@inheritDoc} */
+	public function getHeader($header)
 	{
-		if (empty($field)) {
-			return $this->header;
+		if (empty($header)) {
+			return [];
```

Comment:
Why is it now returning an array? Couldn't the function just return the first entry and we introduce some new function to return the content as an array?

Reply:
I believe it is to match the PSR-7 Response Message interface.